keps/sig-cloud-provider/20190125-out-of-tree-ibm.md

```diff
@@ -40,60 +43,135 @@ status: provisional
 
 - [X] k/enhancements issue in release milestone and linked to KEP (https://github.com/kubernetes/enhancements/issues/671)
 - [ ] KEP approvers have set the KEP status to `implementable`
-- [ ] Design details are appropriately documentedbs
+- [ ] Design details are appropriately documented
 - [ ] Test plan is in place, giving consideration to SIG Architecture and SIG Testing input
 - [ ] Graduation criteria is in place
 - [ ] "Implementation History" section is up-to-date for milestone
 - [ ] Supporting documentation e.g., additional design documents, links to mailing list discussions/SIG meetings, relevant PRs/issues, release notes
 
+## Terms
+
+- **CCM:** Cloud Controller Manager - The controller manager responsible for
+  running cloud provider dependent logic, such as the service and route
+  controllers.
+- **IBM CCM:** IBM cloud provider using CCM out-of-tree cloud provider architecture.
+- **IBM KCM:** IBM cloud provider using KCM in-tree cloud provider architecture.
+  This cloud provider is currently used by IKS.
+- **IKS:** [IBM Cloud Kubernetes Service](https://www.ibm.com/cloud/container-service)
+- **KCM:** Kubernetes Controller Manager - The controller manager responsible
```

Comment:
We are mentioning IBM KCM before explaining the term KCM. Suggest to move this next to CCM i.e. provide Kube terminology before IBM cloud ones. I understand that right now it's alphabetically but mentioning a term before it's usage in doc is recommended for better readability. Also suggest to provider Kube doc hyperlink for KCM.

Reply:
Will do